src/budy/models/bundle_line.py

```
@@ -181,33 +181,39 @@ def ensure_size_s(self):
 
-    def try_valid_quantity(self):
+    def try_valid_quantity(self, bundle=None):
         fixed = False
-        if self.quantity <= 0.0:
-            self.quantity = 0.0
+        quantity = (
+            bundle.merchandise_quantity(self.merchandise) if bundle else self.quantity
+        )
+        if quantity <= 0.0:
+            quantity = 0.0
         if self.merchandise.quantity_hand == None:
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)